appveyor.yml

```diff
@@ -0,0 +1,76 @@
+        Start-Process vcredist_x64.exe -Wait -ArgumentList '/install /passive /norestart'
+        Start-Process msiexec.exe -ArgumentList '/I ModSecurityIIS_2.9.1-RC1-64b.msi /passive'
+        #setup the python virtual env - not a must but still
+        pip install virtualenv
+        virtualenv --version
+        $python_proj_name = "test_crs3"
+        virtualenv $python_proj_name
+        & ".\$python_proj_name\Scripts\activate.ps1"
+        
+
+build_script:
+    - cmd: ./build_release_amd64.bat
+
+before_test:
+    - ps: |
+        #setup the crs repo and the modsec config
+        $modsec_repo = "C:\projects\ModSecurity"
+        & {iisreset /stop}  #need to stop iis before copy of files that are used by it
+        Copy-Item "$modsec_repo\iis\release\amd64\*" -Destination "C:\Windows\System32\inetsrv" -Force
+        #& {iisreset /start}; Start-Service W3SVC
+        #get the crs repo
+        $crs_path = "C:\projects\owasp-modsecurity-crs"
+        git clone -q https://github.com/SpiderLabs/owasp-modsecurity-crs.git $crs_path
+        cd "$modsec_repo\tests"
+        Copy-Item "$crs_path\util\regression-tests\requirements.txt" -Destination "$modsec_repo\tests" -Force
+        #setup the test env with depending python pkgs
+        pip install -r requirements.txt
+        #configure modsec to use crs 3 and set if for the python testing
+        $dbg_file = "$modsec_repo\modsec_debug.log"
+        $modsec_conf = "C:\Program Files\ModSecurity IIS\modsecurity.conf"
+        Add-Content $modsec_conf "SecDebugLog $dbg_file`r`nSecDebugLogLevel 3"
+        $modsec_iis_conf = "C:\Program Files\ModSecurity IIS\modsecurity_iis.conf"
+        Rename-Item -Path "$crs_path\crs-setup.conf.example" -NewName "crs-setup.conf"
+        Clear-Content $modsec_iis_conf
+        #configure modsecurity_iis.conf for crs 3 and for runnig the python tests
+        Add-Content $modsec_iis_conf "Include modsecurity.conf"
+        Add-Content $modsec_iis_conf "SecAction ""id:900005,\`r`n  phase:1,\`r`n  nolog,\`r`n  pass,\`r`n  ctl:ruleEngine=DetectionOnly,\`r`n  ctl:ruleRemoveById=910000,\"
+        Add-Content $modsec_iis_conf "  setvar:tx.paranoia_level=4,\`r`n  setvar:tx.crs_validate_utf8_encoding=1,\`r`n  setvar:tx.arg_name_length=100,\`r`n  setvar:tx.arg_length=400"""
+        Add-Content $modsec_iis_conf "include $crs_path\crs-setup.conf"
+        Add-Content $modsec_iis_conf "include $crs_path/rules/*.conf"
+        & {iisreset}; Start-Service W3SVC
+
+test_script:
+    - ps: |
+        py.test.exe -v CRS_Tests_modsec_dbg.py --rule="$crs_path\util\regression-tests\tests\test.yaml"
+        #running all the tests
+        py.test.exe -q CRS_Tests_modsec_dbg.py --ruledir_recurse="$crs_path\util\regression-tests\tests\" --result-log=res.txt --tb=no --junit-xml=res.xml
+        python Parse_test_res.py res.txt
```

Comment:
how do we get notified whether it was success or failure?

Reply:
the python script return with status code 1 which is abnormal and means that this failed, in the final commit I will add the wafcore alias (or something else) for more distribution